sklearn/metrics/cluster/unsupervised.py

```diff
@@ -133,6 +160,21 @@ def silhouette_samples(X, labels, metric='euclidean', **kwds):
         allowed by :func:`sklearn.metrics.pairwise.pairwise_distances`. If X is
         the distance array itself, use "precomputed" as the metric.
 
+    blockwise: {'auto', True, False}
+        Enables blockwise computation of the distance matrix.
+        If false, the full distance matrix is computed yielding in fast
+        computation but high memory consumption. If True, it computes
+        clusterwise distance matrices, dividing memory consumption by
+        approximately the squared number of clusters. The latter allows
+        parallelization through ``n_jobs`` parameter.
+        Default is 'auto' that chooses the fastest option without
+        consideration for memory consumption.
```

Comment:
@ogrisel , I am getting the windows error 5 access is denied inconsistently (but like 8 out of 10 times) on an Azure VM but not my PC. Is there anything we need to do. I raised an issue with all the details here https://github.com/scikit-learn/scikit-learn/issues/6820 . The VM is Windows Server 2012 R2. Is it something related to VM environment or is there something we can do?